src/basho_bench_worker.erl

```diff
@@ -334,3 +349,27 @@ rate_worker_run_loop(State, Lambda) ->
         ExitReason ->
             exit(ExitReason)
     end.
+
```

Comment:
Made that change, do that sort of thing sometimes as the code gets more and more complicated, Made the changes